static/styles/zulip.css

```diff
@@ -1180,17 +1180,21 @@ td.pointer {
 }
 
 .message_sender i.zulip-icon.bot,
-.popover_info i.zulip-icon.bot {
+.message_sender i.zulip-icon.crown,
+.popover_info i.zulip-icon.bot,
+.popover_info i.zulip-icon.crown {
```

Comment:
It'd be better to use `i.zulip-icon` instead of an explicit `i.zulip-icon.bot` or crown I think, since it might help if we plan to add future icons beside user names in the future